plugins/VstBase/RemoteVstPlugin.cpp

```diff
@@ -2165,7 +2165,10 @@ void RemoteVstPlugin::idle()
 		return;
 	}
 	setProcessing( true );
-	pluginDispatch( effEditIdle );
+	if (!HEADLESS && m_window)
```

Comment:
Technically this could just test `m_window`, since that's the condition we actually care about, and `HEADLESS` implies `!m_window`. But we check `!HEADLESS && m_window` elsewhere in this file, and it's an equivalent condition, so I'm also happy for this to stay as it is.